BetterShards/pom.xml

```diff
@@ -1,101 +0,0 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
-                      http://maven.apache.org/xsd/maven-4.0.0.xsd">
-<modelVersion>4.0.0</modelVersion>
-	<groupId>vg.civcraft.mc.bettershards</groupId>
-	<artifactId>BetterShards</artifactId>
-	<packaging>jar</packaging>
-	<version>1.4.26</version>
-	<name>BetterShards</name>
-	<url>https://github.com/Civcraft/BetterShards</url>		  
-	
-	<dependencies>
-		<dependency>
-			<groupId>vg.civcraft.mc.bettershards</groupId>
-			<artifactId>BetterShardsBukkit</artifactId>
-			<version>1.0.0</version>
-		</dependency>
-	</dependencies>
-	
-	<build>
-		<defaultGoal>clean package</defaultGoal>
-		<directory>../target</directory> <!-- Build it one level up, because it is the one we care about -->
-		<finalName>BetterShards-${project.version}</finalName>
-		<resources>
-			<resource>
-				<targetPath>.</targetPath>
-				<filtering>true</filtering>
-				<directory>..</directory>
-				<includes>
-					<include>LICENSE.txt</include>
-				</includes>
-			</resource>
-			<!-- plugin.yml here to fill in the correct content. -->
-			<resource>
-				<targetPath>.</targetPath>
-				<filtering>true</filtering>
-				<directory>../BetterShardsBukkit</directory>
-				<includes>
-					<include>plugin.yml</include>
-				</includes>
-			</resource>
-			<resource>
-				<targetPath>.</targetPath>
-				<filtering>true</filtering>
-				<directory>../BetterShardsBungee</directory>
-				<includes>
-					<include>bungee.yml</include>
-				</includes>
-			</resource>
-			<!-- config.yml needs its own location so it does not mess with the bukkit plugin. -->
-			<resource>
-				<targetPath>./bungee</targetPath>
```

Comment:
Having this was useful since both bukkit and bungee config are called config.yml. When they are shaded they will both be the same. This should be kept since we are going to be moving away from the civmodcore and including straight config.yml files in the future.